src/AutoRest.CSharp/Common/Input/Inputs.cs

```diff
@@ -161,7 +161,9 @@ internal record InputDictionaryType(string Name, InputType KeyType, InputType Va
 
     internal record InputModelProperty(string Name, string? SerializedName, string Description, InputType Type, bool IsRequired, bool IsReadOnly, bool IsDiscriminator) { }
 
-    internal record InputConstant(object? Value, InputType Type);
+    internal record InputConstant(object? Value, InputType Type) {
+        public InputConstant() : this(null, InputPrimitiveType.String) { }
```

Comment:
I don't think we need default constructor cause we have provided a custom converter.

Reply:
removed.